web/ui/src/features/component/TableHead.tsx

```diff
@@ -0,0 +1,39 @@
+import { useState } from 'react';
+
+interface Props {
+  headers: string[];
+  handleSorting?: (header: string, sortOrder: string) => void;
+}
+
+const TableHead = ({ headers, handleSorting }: Props) => {
+  const [sortField, setSortField] = useState('');
+  const [order, setOrder] = useState('asc');
```

Comment:
I think that the order could be either a bool that you could just toggle with setOrder(!order). But if you want to keep an explicit state then I would suggest using an enum rather than a string